src/components/Header.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-len, @typescript-eslint/ban-types */
```

Comment:
I don't think we should have the ban-types rules set for all these files?

Reply:
The error came up when I upgraded eslint, and I have an issue to track the resolution of the eslint-disables here:
https://github.com/ChainSafe/bls-keygen-app/issues/27